modules/standard/Regexp.chpl

```diff
@@ -597,6 +597,23 @@ record regexp {
     }
   }
 
+  /* did this regular expression compile ? */
+  pragma "no doc"
+  proc ok:bool {
+    compilerWarning("regexp: 'ok' is deprecated; exceptions are used to detect regexp compile errors");
```

Comment:
It's an overloaded/conflated term, but Chapel has errors not exceptions (which to me means stack unwinding).

Reply:
I renamed `exceptions` to `errors`.